.gitignore

```diff
@@ -657,3 +657,5 @@ _book
 author_emails.txt
 # csv files must be converted to parquet files
 *.csv
+super_mega_hubverse_table_v2.parquet
+super_mega_hubverse_table.parquet
```

Comment:
Make a `private` folder in `examples` and `gitignore` that, so that we don't have an expanding list of miscellaneous files.